### PR TITLE
urllib3: add version 1.25.1

### PR DESCRIPTION
**Pull request by grueni**

### CODE REVIEW

File: components/python/urllib3/urllib3-PYVER.p5m [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    value="urllib3 is a powerful, sanity-friendly HTTP client for Python. It brings many critical features that are missing from the Python standard libraries including: 1) Thread safety; 2) Connection pooling; 3) Client-side SSL/TLS verification; 4) File uploads with multipart encoding; 5) Helpers for retrying requests and dealing with HTTP redirects; 6) Support for gzip and deflate encoding; 7) Proxy support for HTTP and SOCKS."
+set name=com.oracle.info.description value="the Python urllib3 module"
+set name=info.classification \
+    value=org.opensolaris.category.2008:Development/Python
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=info.upstream value="Andrey Petrov <andrey.petrov@shazow.net>"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=org.opensolaris.arc-caseid value=PSARC/2015/475
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+#
```
Comment (pyhalov) [inline components/python/urllib3/urllib3-PYVER.p5m:56]: Should 2.7 version also depend on ipaddress? 
Something like 
`$(PYTHON_2.7_ONLY)depend type=require fmri=library/python/ipaddress-$(PYV)`

**Comment by pyhalov** [comment]
Have you tried to build it with setuptools_scm installed?
It seems to create oi-userland git archive during build :)
The following patch heals this:
```
setuptools_scm find_files tries to create archive from whole oi-userland...
--- urllib3-1.25.3/setup.py     2019-05-23 22:01:31.000000000 +0000
+++ urllib3-1.25.3/setup.py     2019-07-30 09:32:48.804949843 +0000
@@ -1,6 +1,8 @@
 #!/usr/bin/env python
 
 from setuptools import setup
+from setuptools_scm import integration
+integration.find_files = lambda p: []
 
 import os
 import re
```

File: components/python/urllib3/Makefile [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+# common targets
+build:		$(BUILD_NO_ARCH)
+
+install:	$(INSTALL_NO_ARCH)
+
+# Tests require Tornado 4.1
+test:		$(NO_TESTS)
+
+# Auto-generated dependencies
```
Comment (pyhalov) [inline components/python/urllib3/Makefile:57]: REQUIRED_PACKAGES also should include library/python/setuptools_scm-{27,34,35} if attached patch is used.

File: components/python/urllib3/urllib3-PYVER.p5m (hunk 1 above)
Comment (grueni) [reply components/python/urllib3/urllib3-PYVER.p5m:56]: No idea at all. I am a complete python moron.

File: components/python/urllib3/Makefile (hunk 2 above)
Comment (grueni) [reply components/python/urllib3/Makefile:57]: Thanks. Patch added.

File: components/python/urllib3/Makefile [hunk 3]
```diff
@@ -57,4 +57,4 @@ test:		$(NO_TESTS)
 REQUIRED_PACKAGES += runtime/python-27
 REQUIRED_PACKAGES += runtime/python-34
 REQUIRED_PACKAGES += runtime/python-35
-
+REQUIRED_PACKAGES += library/python/setuptools_scm-{27,34,35}
```
Comment (pyhalov) [inline components/python/urllib3/Makefile]: Well, I haven't mean literally this :)
Just three separate lines before autogenerated dependencies

File: components/python/urllib3/urllib3-PYVER.p5m (hunk 1 above)
Comment (pyhalov) [reply components/python/urllib3/urllib3-PYVER.p5m:56]: look at requires file, I suppose it depends on it

**Pull request merged**